tests-utils/src/main/scala/tech/beshu/ror/utils/containers/images/DockerImageCreator.scala

```diff
@@ -87,5 +89,11 @@ object DockerImageCreator extends StrictLogging {
         .foldLeft(builder) { case (b, entrypoint) => b.entryPoint(entrypoint.toIO.getAbsolutePath) }
     }
 
+    def setCommandFrom(imageDescription: DockerImageDescription): DockerfileBuilder = {
+      imageDescription
+        .command
+        .foldLeft(builder) { case (b, command) => b.user("elasticsearch").cmd(command) }
```

Comment:
I'm not sure about it. We quietly change the user without setting the previous one back after the cmd